{"name": "Test Get User Details endpoint", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected response content: {'code': '1200', 'message': 'User not authorized!'}\nassert {'code': '120... authorized!'} == {'code': 0, '...ge': 'string'}\n  \n  Differing items:\n  {'code': '1200'} != {'code': 0}\n  {'message': 'User not authorized!'} != {'message': 'string'}\n  Use -v to get more diff", "trace": "@allure.title(\"Test Get User Details endpoint\")\n    def test_get_user_details():\n        user_id = \"12345\"\n        with allure.step(\"Sending GET request to Get User Details endpoint\"):\n            response = requests.get(f\"{BASE_URL}/User/{user_id}\")\n    \n        with allure.step(\"Verifying response status code\"):\n            assert response.status_code in [200, 401], f\"Unexpected response status code: {response.status_code}\"\n    \n        with allure.step(\"Verifying response content\"):\n            if response.status_code == 200:\n                assert \"userId\" in response.json(), f\"Unexpected response content: {response.json()}\"\n            else:\n>               assert response.json() == {\"code\": 0, \"message\": \"string\"}, f\"Unexpected response content: {response.json()}\"\nE               AssertionError: Unexpected response content: {'code': '1200', 'message': 'User not authorized!'}\nE               assert {'code': '120... authorized!'} == {'code': 0, '...ge': 'string'}\nE                 \nE                 Differing items:\nE                 {'code': '1200'} != {'code': 0}\nE                 {'message': 'User not authorized!'} != {'message': 'string'}\nE                 Use -v to get more diff\n\nswagger_test.py:76: AssertionError"}, "steps": [{"name": "Sending GET request to Get User Details endpoint", "status": "passed", "start": 1715438080634, "stop": 1715438081397}, {"name": "Verifying response status code", "status": "passed", "start": 1715438081398, "stop": 1715438081398}, {"name": "Verifying response content", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected response content: {'code': '1200', 'message': 'User not authorized!'}\nassert {'code': '120... authorized!'} == {'code': 0, '...ge': 'string'}\n  \n  Differing items:\n  {'code': '1200'} != {'code': 0}\n  {'message': 'User not authorized!'} != {'message': 'string'}\n  Use -v to get more diff\n", "trace": "  File \"C:\\Users\\Romidamaru\\Desktop\\Основная\\Коды\\swagger_test\\swagger_test.py\", line 76, in test_get_user_details\n    assert response.json() == {\"code\": 0, \"message\": \"string\"}, f\"Unexpected response content: {response.json()}\"\n"}, "start": 1715438081398, "stop": 1715438081400}], "start": 1715438080634, "stop": 1715438081400, "uuid": "6e79b2fa-4965-4403-bdcf-0269877765af", "historyId": "4fa7c63d6a437616fb2dbeda5c89433f", "testCaseId": "4fa7c63d6a437616fb2dbeda5c89433f", "fullName": "swagger_test#test_get_user_details", "labels": [{"name": "suite", "value": "swagger_test"}, {"name": "host", "value": "Romidamaru"}, {"name": "thread", "value": "17336-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "swagger_test"}]}